common/p_hordedefine.cpp

```diff
@@ -213,7 +215,7 @@ size_t P_HordePickDefine(const int current, const int total)
  * @param wantBoss Caller wants a boss.
  */
 bool P_HordeSpawnRecipe(hordeRecipe_t& out, const hordeDefine_t& define,
-                        const bool wantBoss)
+                        const bool wantBoss, MobjTypeTable monsterCounts)
```

Comment:
I think you meant to pass in by const reference here?

Reply:
Yes thank you